syscall.go

```diff
@@ -111,6 +114,62 @@ func unmountShareDir() error {
 	return os.RemoveAll(containerMountDest)
 }
 
+func waitForBlockDevice(deviceName string) (err error) {
+	devicePath := filepath.Join(devPath, deviceName)
+
+	if _, err := os.Stat(devicePath); err == nil {
```

Comment:
Are we sure that because the device is here, it will be ready ?
I am asking because this would mean that we could call into `Stat()` from an infinite loop, until we either find the path, or hit the timeout.